lightning/src/ln/router.rs

```diff
@@ -156,7 +156,7 @@ struct NodeInfo {
 	lowest_inbound_channel_fee_proportional_millionths: u32,
 
 	features: NodeFeatures,
-	last_update: u32,
+	last_update: Option<u32>,
```

Comment:
what's the rationale for moving last_update into an Option as part of this diff? Also, there should be a comment somewhere saying it's a timestamp. 

And also, this will break 18 years from now 😛

Reply:
As otherwise we'll reject a 0 value (as we only accept anything *newer* than the previous value). Also note that it is *not* a timestamp, the spec only recommends that you use a timestamp. See-also #493. I added a comment noting that None implies we've not heard any updates yet.